### DISCUSSION TITLE
Download HAR file using seleniumbase

### DISCUSSION THREAD

**Discussion by db-dhananjayb** [seleniumbase/SeleniumBase#2119] [Q&A] [answered]
At the end of selenium script, I want to download HAR file from the browser.
Does seleniumbase has a native way to do this?
While searching I came across  this : https://github.com/seleniumbase/SeleniumBase/discussions/1142
However it seems it uses proxy and I want to do this without proxy. Any ideas/suggestions?
Thanks in advance.

**Comment by mdmintz** [comment, marked as answer]
You can generate your own HAR file from the network logs.

See https://github.com/seleniumbase/SeleniumBase/blob/master/examples/uc_cdp_events.py for an example.
